src/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
So these `Directory.Build.props` and `*.targets` files let us describe per-directory behavior? Neat.